config/routes.rb

```diff
@@ -2425,7 +2425,17 @@
         index
         show
         show_list
-      ]
+        new
+      ],
+      :post => %w[
+        button
```

Comment:
`button` route and others are probably not needed, can you verify.

Reply:
Verified.